src/middleware/checkAdmin.js

```diff
@@ -0,0 +1,14 @@
+const checkAdmin = (req, res, next) => {
+  const userRole = req.userData.role;
+
+  if (userRole !== 'super-admin') {
```

Comment:
use the `or` operator instead of nesting ifs

Reply:
I tried to use it before but it was not working as required. But let me find another way

Reply:
> `or` operator

I used `and` operator and worked

Reply:
Done